starknet/src/authenticators/eth_tx.cairo

```diff
@@ -3,6 +3,9 @@ use sx::types::{Strategy, IndexedStrategy, Choice};
 
 #[starknet::interface]
 trait IEthTxAuthenticator<TContractState> {
```

Comment:
any reason these comments are different format to others?

Reply:
lazyness :D  015174552e7f75c6882639b367d1cd54b8d6fb1b